src/store/index.ts

```diff
@@ -1,9 +1,20 @@
 import Vue from "vue";
 import Vuex from "vuex";
+import VuexPersist from "vuex-persist";
 
 Vue.use(Vuex);
 
+const vuexLocalStorage = new VuexPersist({
+  key: "vuex", // The key to store the state on in the storage provider.
+  storage: window.localStorage, // or window.sessionStorage or localForage
```

Comment:
Let's change this to sessionStorage instead.  Also, let's make sure and clear sessionStorage as part of the logout action, b/c the logout takes the user back to the login page and the session will be persisted.

Reply:
Using window.sessionStorage might cause some collision when you different branches of the application since the session can live for various days in the browser. and some very unexpected bugs. 

But if that is what you want let make it happen.

Reply:
Second disclaimer: Cleaning the session on log out means that any other data of the state of the application will be also clear out. 

this last one I guess is what you want to happen right?

Reply:
done

Reply:
Thanks, We don't localStorage to persist data as someone else most likely could log into the same workstation and create another portoflio, application, etc.,  I suspect in the future we will remove what is applicable from sessionStorage that pertains to our app on the logout action.